scripts/check-new-images-exist.sh

```diff
@@ -7,20 +7,36 @@ fi
 
 echo "Checking for new images in commit(s) against ${DIFF_CHECK}"
 
-NEW_IMAGES=$(git diff -U0 $DIFF_CHECK -- images-list | tail -n +5 | grep -v ^@@ | grep -v ^- | cut -d+ -f2 | awk '{ print $1":"$3 }')
+NEW_IMAGES=$(git diff -U0 $DIFF_CHECK -- images-list | tail -n +5 | grep -v ^@@ | grep -v ^- | cut -d+ -f2 | awk '{ print $1":"$3","$2 }')
 
 if [ -z "${NEW_IMAGES}" ]; then
   echo "Could not find new images in commit(s) against ${DIFF_CHECK}"
   exit 0
 fi
 
-echo "Found new images in commit(s) against ${DIFF_CHECK}: ${NEW_IMAGES}"
+echo -e "Found new images in commit(s) against ${DIFF_CHECK}:\n${NEW_IMAGES}"
 
 for NEW_IMAGE in $NEW_IMAGES; do
-  echo "Checking if image ${NEW_IMAGE} exists"
-  if ! skopeo inspect --retry-times=3 "docker://${NEW_IMAGE}" >/dev/null; then
-    echo "Image ${NEW_IMAGE} does not exist"
+  SOURCE_IMAGE=$(echo "${NEW_IMAGE}" | cut -d, -f1)
+  TARGET_IMAGE=$(echo "${NEW_IMAGE}" | cut -d, -f2)
+  if [ $(awk '$2 == "'"$TARGET_IMAGE"'" { print $2 }' images-list | wc -l) -eq 1 ]; then
+    echo "${TARGET_IMAGE} is a new target repository"
+    TARGET_NAMESPACE=$(echo "${TARGET_IMAGE}" | cut -d/ -f1)
+    TARGET_REPOSITORY=$(echo "${TARGET_IMAGE}" | cut -d/ -f2)
+    echo "Checking if Docker Hub namespace ${TARGET_NAMESPACE} and repository ${TARGET_REPOSITORY} exists"
+    if curl -H "Content-Type: application/json" --silent --fail "https://hub.docker.com/v2/namespaces/${TARGET_NAMESPACE}/repositories/${TARGET_REPOSITORY}/" > /dev/null; then
```

Comment:
I think you want this to be an Accept header; the Content-Type header is part of the response.
```suggestion
    if curl -H "Accept: application/json" --silent --fail "https://hub.docker.com/v2/namespaces/${TARGET_NAMESPACE}/repositories/${TARGET_REPOSITORY}/" > /dev/null; then
```

Reply:
Yeah I miss copy pasted it, but it doesn't really matter as we fail on response so I removed it.